src/rust/lib/frp/src/lib.rs

```diff
@@ -115,24 +111,24 @@ mod tests {
     use crate as frp;
     use crate::*;
 
-    #[test]
-    fn counter() {
-        frp::new_network! { network1
-            def source = source();
-        }
-        frp::new_network! { network2
-            def count = source.count();
-        }
-        assert_eq!(count.value(),0);
-        source.ping();
-        assert_eq!(count.value(),1);
-        source.ping();
-        assert_eq!(count.value(),2);
-        mem::drop(network1);
-        source.ping();
-        assert_eq!(count.value(),2);
-        mem::drop(network2);
-        source.ping();
-        assert_eq!(count.value(),0);
-    }
+    // #[test]
+    // fn counter() {
+    //     frp::new_network! { network1
+    //         def source = source();
+    //     }
+    //     frp::new_network! { network2
+    //         def count = source.count();
+    //     }
+    //     assert_eq!(count.value(),0);
+    //     source.ping();
+    //     assert_eq!(count.value(),1);
+    //     source.ping();
+    //     assert_eq!(count.value(),2);
+    //     mem::drop(network1);
+    //     source.ping();
+    //     assert_eq!(count.value(),2);
+    //     mem::drop(network2);
+    //     source.ping();
+    //     assert_eq!(count.value(),0);
+    // }
```

Comment:
Another test which was commented, but I don't know why.

Reply:
As agreed via phone - tests will be added in next prs.